server/routes/stories.js

```
@@ -215,35 +215,75 @@ router.post('/create', authRequired, async (req, res) => {
  *       201:
  *         description: Story uploaded successfully.
  */
-// POST /api/v1/stories/upload - Upload user story
-router.post('/upload', authRequired, async (req, res) => {
+// POST /api/v1/stories/upload - Upload user story (Live Writer)
+router.post('/upload', authRequired, fileUploadOptions.fields([{ name: 'coverImage', maxCount: 1 }]), async (req, res) => {
```

Biome (2.4.4)

[error] This variable is used before its declaration.

(lint/correctness/noInvalidUseBeforeDeclaration)